src/MUIDataTable.js

```diff
@@ -41,6 +41,11 @@ const TABLE_LOAD = {
   UPDATE: 2,
 };
 
+// Wrapper Component to keep the rawValue
+function FilterValue(props) {
```

Comment:
Can we move this to line 669? since it isn't referenced anywhere else I would rather have it in the class

Reply:
This is a React functional component, I think it its quite unusual to have that as a property on a class, isn't it?

Reply:
Let's separate it out in a different file then

Reply:
Actually don't worry about this